internal/resources/sourcesecret/data_sourcesecret_test.go

```diff
@@ -41,39 +41,6 @@ func TestAcceptanceForSourceSecretDataSource(t *testing.T) {
 				Config: testConfig.getTestSourceSecretDataSourceBasicConfigValue(commonscope.ClusterScope, spec.SSHKey),
 				Check:  testConfig.checkSourceSecretDataSourceAttributes(),
 			},
-			{
-				Config: testConfig.getTestSourceSecretDataSourceBasicConfigValue(commonscope.ClusterScope, spec.SSHKey, WithKnownhosts("somehostes")),
-				Check:  testConfig.checkSourceSecretDataSourceAttributes(),
-			},
-			{
-				Config: testConfig.getTestSourceSecretDataSourceBasicConfigValue(commonscope.ClusterGroupScope, spec.SSHKey),
-				Check:  testConfig.checkSourceSecretDataSourceAttributes(),
-			},
-			{
-				Config: testConfig.getTestSourceSecretDataSourceBasicConfigValue(commonscope.ClusterGroupScope, spec.SSHKey, WithKnownhosts("somehostes")),
-				Check:  testConfig.checkSourceSecretDataSourceAttributes(),
-			},
-		},
-	},
-	)
-
-	t.Log("source secret data source acceptance test complete for SSH credential type")
-
-	// Test case for source secret data source with Username/Password credential type.
```

Comment:
Why are we removing the tests?

Reply:
tests which I've removed they were covering one specific spec type which can be tests through update scenario that's why I've merged those tests in to one.